stories/core/Table.stories.tsx

```diff
@@ -788,6 +788,100 @@ RowInViewport.argTypes = {
   data: { control: { disable: true } },
 };
 
+export const DisabledRows: Story<TableProps> = (args) => {
+  const { columns, data, ...rest } = args;
+
+  const isRowDisabled = useCallback(
+    (rowData: { name: string; description: string }) => {
+      return rowData.name === 'Name2';
+    },
+    [],
+  );
+
+  const tableColumns = useMemo(
+    () => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+            maxWidth: 200,
+          },
+        ],
+      },
+      {
+        id: 'click-me',
+        Header: 'Click',
+        width: 100,
+        // Manually handling disabled state in custom cells
+        Cell: (props: CellProps<{ name: string; description: string }>) => (
+          <>
+            {isRowDisabled(props.row.original) ? (
+              <>Click me!</>
+            ) : (
+              <a
+                className='iui-anchor'
+                onClick={action(props.row.original.name)}
```

Comment:
we should not try to disable everything ourselves in table? like eat all the events, etc...

Reply:
Do we really want? What if there is a dropdown menu that has item to enable that row. Or even this example, idk how I could override it.

Reply:
Well, I think by design, if row is disabled, it should not be interactive in any case? but maybe I agree, lets leave it... Just then need to be sure and inform users to handle it (I bet they wont!, that is my concern.. and all those menus will be still openable in disabled row).

Reply:
Resolved?